src/applications/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -168,7 +173,7 @@ export const uiSchema = {
       emailAddress: {
         'ui:title': 'Email address',
         'ui:errorMessages': {
-          pattern: 'Please put your email in this format x@x.xxx'
+          pattern: 'Email format must be at least x@x.xx'
```

Comment:
This could use some revision.

Reply:
For reference, the actual validation is that the email must have a minimum of `x@x.xx`

Reply:
Sure, thanks @peggygannon !